src/i18n/es.js

```diff
@@ -20,8 +20,8 @@ export const messages_es = {
   'buttons.add_content': `Agregar contenido`,
   'buttons.add_display_condition': `Añadir requisito de pantalla`,
   'buttons.add_field': `Añadir campo`,
-  'buttons.add_new_field': `Añadir Nuevo Campo`,
-  'buttons.add_new_item': `Añadir nuevo elemento`,
+  'buttons.add_new_field': `Añadir un nuevo campo`,
+  'buttons.add_new_item': `Añadir un nuevo elemento`,
```

Comment:
No se si en estos casos el `un` es necesario.

```suggestion
  'buttons.add_new_field': `Añadir nuevo campo`,
  'buttons.add_new_item': `Añadir nuevo elemento`,
```

@GGossweiler que opinás?